python/auditlog/templates/auditlog/_logentry_list.html

```diff
@@ -0,0 +1,63 @@
+{% if object_list.exists %}
+<table id="auditlog" class="listtable full-width">
+<thead>
+<tr>
+    <th>Timestamp</th>
+    <th>Actor</th>
+    <th>Verb</th>
+    <th>Object</th>
+    <th>Target</th>
+    <th>Summary</th>
+</tr>
+</thead>
+</table>
+{% else %}
+No auditlog for {{ auditmodel }}.
+{% endif %}
+<script type="text/javascript">
+    require(['libs/datatables.min'], function(){
+        $('.latest-changes-heading').click(function(){
+            $('.latest-changes-content').slideToggle();
+            $('.latest-changes-heading i').toggleClass('fa-caret-square-o-right fa-caret-square-o-down');
+        });
+
+        var dt_config = {
+            ajax: {
+                url: '/api/1/auditlog/?subsystem=portadmin',
```

Comment:
This file is used both for portadmin-only entries *and* all entries, to capture what doesn't change.

Wouldn't this limit the the list to only ever fetching portadmin logs?